cmd/envs.go

```diff
@@ -265,7 +265,19 @@ func createPipelineEnv(client api.Client, stackId int64, name, providerName, reg
 }
 
 func createPreviewEnv(client api.Client, stackId int64, name string) error {
-	env, err := client.PreviewEnvs().Create(stackId, &api.CreatePreviewEnvInput{Name: name})
+	newEnv, err := client.Environments().Create(stackId, &types.Environment{
+		OrgName: client.Config.OrgName,
+		StackId: stackId,
+		Name:    name,
+		Type:    types.EnvTypePreview,
+	})
+	if err != nil {
+		return fmt.Errorf("error creating preview environment: %w", err)
+	} else if newEnv == nil {
+		return fmt.Errorf("unable to create preview environment")
+	}
+
+	env, err := client.PreviewEnvs().Update(stackId, newEnv.Id, &api.UpdatePreviewEnvInput{Name: &name})
```

Comment:
Not sure I understand this, why would we update the preview environment immediately after creating it?

Reply:
That's my bad. When I updated go-api-client library, the `Create` function was missing, so I switched over to `Update`. Then, I realized that wasn't going to create the env. After creating the env, I didn't come back around and remove it.

Dropping this API call.